int32 id
string topic_name
time stamp
int32[] channels
string model_path
---
int32 id
sensor_msgs/Image spec
string result